httpobs/database/schema.sql

```diff
@@ -111,3 +111,43 @@ GRANT UPDATE (cookies) ON sites TO httpobsapi;
 /*
 ALTER TABLE scans ADD COLUMN likelihood_indicator VARCHAR NULL;
 */
+
+/* Update to frequently refresh latest_scans */
+/*
+GRANT SELECT ON latest_scans TO httpobsapi;
```

Comment:
Nope, just a reminder that this was done to a live system. I'll probably go through and remove these eventually.